src/tilemap_layer.cpp

```diff
@@ -566,7 +578,7 @@ BitmapRef TilemapLayer::GenerateAutotiles(int count, const std::map<uint32_t, Ti
 				rect.x = (x * 2 + i) * (TILE_SIZE/2);
 				rect.y = (y * 2 + j) * (TILE_SIZE/2);
 
-				tiles->Blit((dst.x * 2 + i) * (TILE_SIZE / 2), (dst.y * 2 + j) * (TILE_SIZE / 2), *chipset, rect, 255);
+				tiles->BlitFast((dst.x * 2 + i) * (TILE_SIZE / 2), (dst.y * 2 + j) * (TILE_SIZE / 2), *chipset, rect, 255);
```

Comment:
There is no game using transparent auto tiles but I have to flag this for an accuracy test

Reply:
I don't really know why but semi-transparent autotiles still render properly :thinking:

Reply:
This is creating a secondary bitmap to store the generated autotiles. Since the bitmap is previously empty, we can fast blit the autotiles onto it.